README.md

```diff
@@ -1,2 +1,13 @@
 # cli-games
-Games in the terminal
+## About this project:
+This repository contains a collection of cli (command line iterface games). Please feel free to have a look through the code of each game (usually just a single file of code) after playing, this will help you understand the logic that goes behind making such games.
+## How to play ?
+1. Clone this repo
+2. Install required environment (node for js and python interpreter for python)
+3. Execute the game file inside terminal (like node game.js or python game.py)
```

Comment:
Since we can't play all games by node game.js or python game.py it'd better to ask the users to refer the folder of each game forore instructions

Reply:
Also can you add **setting the environment** in a different section so it'd get more attention